roktux/src/main/java/com/rokt/roktux/CacheConfig.kt

```diff
@@ -0,0 +1,3 @@
+package com.rokt.roktux
+
+class CacheConfig(val viewStateChange: ((RoktViewState) -> Unit), val viewState: RoktViewState? = null)
```

Comment:
nit: Is the order important here? 
For iOS I think we will use the viewStateChange last.

Reply:
It is not important, however I've updated it to match the convention. Also refactored the class name.